scripts/pre_commit_linter.py

```diff
@@ -102,12 +102,47 @@
         'regexp': r"\b(ddescribe|fdescribe)\(",
         'message': "In tests, please use 'describe' instead of 'ddescribe'"
                    "or 'fdescribe'",
-        'excluded_files': ()
+        'excluded_files': (),
+        'excluded_dirs': ()
     },
     {
         'regexp': r"\b(iit|fit)\(",
         'message': "In tests, please use 'it' instead of 'iit' or 'fit'",
-        'excluded_files': ()
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': r"templateUrl: \'",
+        'message': "The directives must be directly referenced.",
+        'excluded_files': (
+            'core/templates/dev/head/pages/exploration_editor/'
```

Comment:
I understand the issue with the feedback popup directive, but why is this one excluded? Can it be fixed, instead?

Reply:
Here, A variable in html is used to include or not include the modal. I think there's another thread that was going on for removing all jinja from the html which should handle this variable.

![screenshot from 2017-12-11 09-22-48](https://user-images.githubusercontent.com/9693472/33815105-e8bfc2b4-de54-11e7-9a36-1c8e75f93392.png)

Reply:
I think that's a different issue, actually. The Jinja stuff doesn't have to be removed in this PR, but the modal HTML should still be moved to a separate file and the existing patterns should be followed.

Reply:
Ok, that means I should move the modal code along with the condition ({% %}) to a new file, right?

I think It may not work that way as jinja would not compile when the file is called through UrlInterpolationService. I am not sure though.

Can you clarify please?

Reply:
I don't think you need to move the ({% %}), just the stuff inside it.

Reply:
done.